### PR TITLE
Add check_openmanage command to ITL.

### PR DESCRIPTION
**Pull request by sahnd**
This will help monitoring Dell PowerEdge servers hardware with standard itl command definition.

### CODE REVIEW

File: itl/plugins-contrib.d/hardware.conf [hunk 1]
```diff
@@ -96,3 +96,110 @@ object CheckCommand "hpasm" {
...
+			description = "Check everything, even log content"
+		}
+		"--blacklist" = {
+			value = "$openmanage_blacklist$"
+			repeat_key = true
+			description = "Blacklist missing and/or failed components"
+		}
+		"--check" = {
+			value = "$openmanage_check$"
+			description = "Fine-tune which components are checked
```
Comment (mcktr) [inline itl/plugins-contrib.d/hardware.conf]: The closing quotation mark is missing.

**Comment by mcktr** [comment]
Wow that was fast :) Can you please squash your commits into a single one?

**Comment by sahnd** [comment]
Hope this is correct now. :)

**Comment by mcktr** [comment]
Thanks! Looks good to me. :+1:

File: itl/plugins-contrib.d/hardware.conf [hunk 2]
```diff
@@ -96,3 +96,111 @@ object CheckCommand "hpasm" {
...
+		"--ipv6" = {
+			set_if = "$openmanage_ipv6$"
+			description = "Use IPv6 instead of IPv4 [default=no]"
+		}
+		"--legacy-perfdata" = {
+			set_if = "$openmanage_legacyperfdata$"
+			description = "legacy performance data output"
+		}
+		"--no-storage" = {
+			set_if = "$openmanage_no-storage$"
```
Comment (lazyfrosch) [inline itl/plugins-contrib.d/hardware.conf]: Please avoid the dashes `-` and use underscores `_` for the whole variable.

File: itl/plugins-contrib.d/hardware.conf [hunk 3]
```diff
@@ -96,3 +96,111 @@ object CheckCommand "hpasm" {
...
+		"--protocol" = {
+			value = "$openmanage_protocol$"
+			description = "SNMP protocol version [default=2c]"
+		}
+		"--short-state" = {
+			set_if = "$openmanage_shortstate$"
+			description = "Prefix alerts with alert state abbreviated"
+		}
+		"--show-blacklist" = {
+			set_if = "$openmanage_show-blacklist$"
```
Comment (lazyfrosch) [inline itl/plugins-contrib.d/hardware.conf]: Please avoid the dashes `-` and use underscores `_` for the whole variable.

File: itl/plugins-contrib.d/hardware.conf [hunk 4]
```diff
@@ -96,3 +96,111 @@ object CheckCommand "hpasm" {
...
+		"--tcp" = {
+			set_if = "$openmanage_tcp$"
+			description = "Use TCP instead of UDP [default=no]"
+		}
+		"--timeout" = {
+			value = "$openmanage_timeout$"
+			description = "Plugin timeout in seconds [default=30]"
+		}
+		"--vdisk-critical" = {
+			set_if = "$openmanage_vdisk-critical$"
```
Comment (lazyfrosch) [inline itl/plugins-contrib.d/hardware.conf]: Please avoid the dashes `-` and use underscores `_` for the whole variable.

**Comment by sahnd** [comment]
Changed every dash to underscore. Changed some vars to be more in sync with parameters.

Hope this is fine now.

**Comment by lazyfrosch** [comment]
Awesome, thank you!

**Comment by lazyfrosch** [comment]
Travis bugged out, but this doesn't matter for this PR...

**Pull request merged**